### PR TITLE
Add Admonitions to Maintain Content

### PR DESCRIPTION
**Pull request by salmad3**
Part of https://github.com/w3f/polkadot-wiki/issues/3133

### CODE REVIEW

File: docs/maintain/maintain-guides-how-to-nominate-polkadot.md [hunk 1]
```diff
@@ -6,10 +6,11 @@ description: Steps on how to nominate on Polkadot.
 slug: ../maintain-guides-how-to-nominate-polkadot
 ---
 
-> The following information applies to the Polkadot network. If you want to nominate on Kusama,
-> check out the
-> [Kusama guide](./kusama/maintain-guides-how-to-nominate-kusama.md)
-> instead.
+:::info The following information applies to the Polkadot network
```
Comment (profradha) [inline docs/maintain/maintain-guides-how-to-nominate-polkadot.md]:
```suggestion
:::info 
```

File: docs/maintain/maintain-guides-how-to-nominate-polkadot.md [hunk 2]
```diff
@@ -6,10 +6,11 @@ description: Steps on how to nominate on Polkadot.
 slug: ../maintain-guides-how-to-nominate-polkadot
 ---
 
-> The following information applies to the Polkadot network. If you want to nominate on Kusama,
-> check out the
-> [Kusama guide](./kusama/maintain-guides-how-to-nominate-kusama.md)
-> instead.
+:::info The following information applies to the Polkadot network 
+
+If you want to nominate on Kusama, check out the [Kusama guide](./kusama/maintain-guides-how-to-nominate-kusama.md) instead.
```
Comment (profradha) [inline docs/maintain/maintain-guides-how-to-nominate-polkadot.md]:
```suggestion
The following information applies to the Polkadot network. If you want to nominate on Kusama, check out the [Kusama guide](./kusama/maintain-guides-how-to-nominate-kusama.md) instead.
```

File: docs/maintain/maintain-guides-how-to-nominate-polkadot.md [hunk 3]
```diff
@@ -94,8 +95,12 @@ earn compound interest.
 
 ![Payout account selection dropdown with the custom account option highlighted](../assets/payout/01.png)
 
-> These concepts have been further explained in Polkadot's
-> [UI Walkthrough Video](https://youtu.be/FCXC0CDhyS4?t=219)
+:::note Explainer video on nominator account actions using the PolkadotJS UI
```
Comment (profradha) [inline docs/maintain/maintain-guides-how-to-nominate-polkadot.md]:
```suggestion
:::note 
```

File: docs/maintain/maintain-guides-how-to-stop-validating.md [hunk 4]
```diff
@@ -30,10 +30,14 @@ To chill your validator or nominator, call the `staking.chill()` extrinsic. See
 Purging the validator's session keys removes the key reference to your stash. This can be done
 through the `session.purgeKeys()` extrinsic with the controller account.
 
-> NOTE: **If you skip this step, you will not be able to reap your stash account**, and you will
-> need to rebond, purge the session keys, unbond, and wait the unbonding period again before being
-> able to transfer your tokens. See [Unbonding and Rebonding](maintain-guides-how-to-unbond.md) for
-> more details.
+:::caution If you skip this step, you will not be able to reap your stash account
```
Comment (profradha) [inline docs/maintain/maintain-guides-how-to-stop-validating.md]:
```suggestion
:::caution 
```

File: docs/maintain/maintain-guides-how-to-stop-validating.md [hunk 5]
```diff
@@ -30,10 +30,14 @@ To chill your validator or nominator, call the `staking.chill()` extrinsic. See
...
+
+You will also need to rebond, purge the session keys, unbond, and wait the unbonding period
```
Comment (profradha) [inline docs/maintain/maintain-guides-how-to-stop-validating.md]:
```suggestion
**If you skip this step, you will not be able to reap your stash account**, and you will also need to rebond, purge the session keys, unbond, and wait the unbonding period 
```

File: docs/maintain/maintain-guides-how-to-validate-polkadot.md [hunk 6]
```diff
@@ -6,13 +6,14 @@ description: The fundamentals for running a Polkadot validator.
 slug: ../maintain-guides-how-to-validate-polkadot
 ---
 
-> The following information applies to the Polkadot network. If you want to set up a validator on
-> Kusama, check out the
-> [Kusama guide](kusama/mirror-maintain-guides-how-to-validate-kusama.md)
-> instead.
+:::info The following information applies to the Polkadot network
```
Comment (profradha) [inline docs/maintain/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
:::info 
```

File: docs/maintain/maintain-guides-how-to-validate-polkadot.md [hunk 7]
```diff
@@ -6,13 +6,14 @@ description: The fundamentals for running a Polkadot validator.
...
+
+If you want to set up a validator on Kusama, check out the [Kusama guide](kusama/mirror-maintain-guides-how-to-validate-kusama.md) instead.
```
Comment (profradha) [inline docs/maintain/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
The following information applies to the Polkadot network. If you want to set up a validator on Kusama, check out the [Kusama guide](kusama/mirror-maintain-guides-how-to-validate-kusama.md) instead.
```

File: docs/maintain/maintain-guides-how-to-validate-polkadot.md [hunk 8]
```diff
@@ -544,7 +594,7 @@ you are not selected for the validator set in a particular era. However, it may
 increase the number of DOT staked or seek out nominators for your validator in order to join the
 validator set.
 
-**Congratulations!** If you have followed all of these steps, and been selected to be a part of the
+**Congratulations**, if you have followed all of these steps, and been selected to be a part of the
```
Comment (profradha) [inline docs/maintain/maintain-guides-how-to-validate-polkadot.md]: Congratulations with an exclamation mark is valid. Not sure why you wanted to change it. A comma beside congratulations is not reading well https://dictionary.cambridge.org/dictionary/english/congratulation

```suggestion
**Congratulations!** If you have followed all of these steps, and been selected to be a part of the
```

File: docs/maintain/maintain-guides-validator-payout.md [hunk 9]
```diff
@@ -182,12 +190,16 @@ their validator to "reimburse" themselves for the cost of running a validator no
 all rewards are shared based on the stake behind each validator. This includes the stake of the
 validator itself, plus any stake bonded by nominators.
 
-> **NOTE:** Validators set their preference as a percentage of the block reward, _not_ an absolute
-> number of DOT. Polkadot's block reward is based on the _total_ amount at stake, with the reward
-> peaking when the amount staked is at 50% of the total supply. The commission is set as the amount
-> taken by the validator; that is, 0% commission means that the validator does not receive any
-> proportion of the rewards besides that owed to it from self-stake, and 100% commission means that
-> the validator operator gets all rewards and gives none to its nominators.
+:::info Validators set their preference as a percentage
```
Comment (profradha) [inline docs/maintain/maintain-guides-validator-payout.md]:
```suggestion
:::info 
```

**Review by DrW3RK — CHANGES_REQUESTED** [review]

File: docs/maintain/maintain-guides-how-to-validate-polkadot.md (hunk 8 above)
Comment (salmad3) [reply docs/maintain/maintain-guides-how-to-validate-polkadot.md]: We should avoid exclamation marks in technical docs, and don't necessarily need "Congratulations" here, but in terms of the page's writing style it should be reconsidered when the page is reviewed. Will keep as is for now

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**